src/test/java/com/exasol/ScalarFunctionsIT.java

```diff
@@ -14,12 +16,38 @@
 class ScalarFunctionsIT extends AbstractLuaVirtualSchemaIT {
     @Test
     void testIproc() {
-        final String sourceSchemaName = "IPROC_SCHEMA";
-        final Schema sourceSchema = createSchema(sourceSchemaName);
+        final Schema sourceSchema = createSchema("IPROC_SCHEMA");
         sourceSchema.createTable("T", "C1", "BOOLEAN").insert(true).insert(false);
         final VirtualSchema virtualSchema = createVirtualSchema(sourceSchema);
-        final User user = createUserWithVirtualSchemaAccess("UP_USER", virtualSchema);
-        assertRlsQueryWithUser("SELECT IPROC() FROM " + getVirtualSchemaName(sourceSchemaName) + ".T", user,
+        final User user = createUserWithVirtualSchemaAccess("IPROC_USER", virtualSchema);
+        assertRlsQueryWithUser("SELECT IPROC() FROM " + getVirtualSchemaName(sourceSchema) + ".T", user,
                 table().row(0).row(0).matches(NO_JAVA_TYPE_CHECK));
```

Comment:
`.row(0)` is called twice. What does this mean?

Reply:
[IPROC](https://docs.exasol.com/sql_references/functions/alphabeticallistfunctions/iproc.htm) tells you on which cluster node a row resides. We have two rows on a single-node cluster in this test, so the node index is zero for every row.